graylog2-web-interface/src/components/notifications/Notification.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import DOMPurify from 'dompurify';
+
+import { Alert, Button } from 'components/bootstrap';
+import { RelativeTime, Icon, Spinner } from 'components/common';
+import NotificationsFactory from 'logic/notifications/NotificationsFactory';
+import type { NotificationType } from 'stores/notifications/NotificationsStore';
+import { NotificationsActions, NotificationsStore } from 'stores/notifications/NotificationsStore';
+import { useStore } from 'stores/connect';
+
+type Props = {
+  notification: NotificationType,
+};
+
+const StyledButton = styled(Button)`
+  float: right;
+`;
+
+const StyledAlert = styled(Alert)(({ theme }) => css`
+  margin-top: 10px;
+
+  i {
+    color: ${theme.colors.gray[10]};
+  }
+
+  form {
+    margin-bottom: 0;
+  }
+`);
+
+const NotificationHead = styled.h3`
+  margin-bottom: 5px;
+`;
+
+const NotificationTimestamp = styled.span(({ theme }) => css`
+  margin-left: 3px;
+  font-size: ${theme.fonts.size.small};
+`);
+
+const _sanitizeDescription = (description) => {
+  return DOMPurify.sanitize(description);
+};
+
+/* eslint-disable react/no-danger */
```

Comment:
Can we only disable it on one specific line, or at least place this above the return?